CONTRIBUTING.md

```diff
@@ -115,3 +115,34 @@ If it does, please add the new generic class to the `_need_generic` list in the
 We only add hints for private attributes when it has some demonstrated real-world use case.
 That means from a third-party package or some well described snippet for a project.
 This rule helps us avoid tying in too closely to Django’s undocumented internals.
+
+
+## Releasing `django-stubs`
+
+1. Open a pull request that updates `setup.py`, `ext/setup.py` and `README.md`
+   (anyone can open this PR, not just maintainers):
+
+    - Increase `version=` value within `setup(...)` call in **both** files. The versions must be in sync.
+    - Update `django-stubs-ext>=>=` dependency in root `setup.py` to the same version number.
+    - Version number `major.minor.patch` is formed as follows:
+
+      `major.minor` version must match newest supported Django release.
+
+      `patch` is sequentially increasing for each stubs release. Reset to `0` if `major.minor` was updated.
+
+    - Use pull request title "Version x.y.z release" by convention.
```

Comment:
I notice now that flaeppe has been using PR title "Prepare for a release \`5.0.1`". I don't have strong feelings either way. Except for the fact that the "a" there seems ungrammatical and can be omitted.